scripts/tests/calibnet_wallet_rpc_check.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+# This script checks some RPC methods of the forest node.
+# It requires both `forest` and `forest-wallet` to be in the PATH.
+
+set -euxo pipefail
+
+source "$(dirname "$0")/harness.sh"
+
+
+usage() {
+  echo "Usage: $0 <PRELOADED_WALLET_STRING>"
+  exit 1
+}
+
+if [ -z "$1" ]
+  then
+    usage
+fi
+
+echo "$1" > preloaded_wallet.key
+
+forest_init
+
+: Begin Filecoin.MarketAddBalance test
+
+FOREST_URL='http://127.0.0.1:2345/rpc/v1'
+
+$FOREST_WALLET_PATH --remote-wallet import preloaded_wallet.key
+
+# The preloaded address
+REMOTE_ADDR=$($FOREST_WALLET_PATH --remote-wallet list | tail -1 | cut -d ' ' -f1)
+
+# Amount to add to the Market actor (in attoFIL)
+FIL_AMT="23"
+
+JSON=$(curl -s -X POST "$FOREST_URL" \
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/json' \
+  --header "Authorization: Bearer $ADMIN_TOKEN" \
+  --data "$(jq -n --arg addr "$REMOTE_ADDR" --arg amt "$FIL_AMT" '{jsonrpc: "2.0", id: 1, method: "Filecoin.MarketAddBalance", params: [$addr, $addr, $amt]}')")
+
+echo "$JSON"
+
+if [[ $(echo "$JSON" | jq -e '.result') == "null" ]]; then
+  echo "Error while sending message."
+  exit 1
+fi
+
+MSG_CID=$(echo "$JSON" | jq -r '.result["/"]')
+echo "Message cid: $MSG_CID"
+
+# Try 10 times.
+for i in {1..10}
+do
+  sleep 30s
```

Comment:
Should we rather have more retries and less sleeps? This would make the test pass faster.

Reply:
This duration was chosen because Filecoin has a block time of 30 seconds.

I will see what I can do to speed up the test.

Reply:
Usually, we won't end up at the block boundary, so it is possible that the message will be there in less than 30 seconds (or less than 60 seconds, e.g., after 35s). Is it correct?

Reply:
Yes, correct. I've moved to a 5s turnaround time.

Reply:
10m 59s ➡️  9m 55s. Still most of the time is dominated by the setup.

Reply:
Can we re-use the existing wallet check? That would help a lot with the setup and burn less resources in process. https://github.com/ChainSafe/forest/blob/main/scripts/tests/calibnet_wallet_check.sh

Reply:
Right.